cfme/storage/volume_backup.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+import attr
+import random
+
+from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic_manageiq import (
+    Accordion,
+    BaseEntitiesView,
+    BreadCrumb,
+    ItemsToolBarViewSelector,
+    ManageIQTree,
+    SummaryTable
+)
+from widgetastic_patternfly import BootstrapSelect, Button, Dropdown, FlashMessages
+from widgetastic.widget import View, Text
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.common import TagPageView, WidgetasticTaggable
+from cfme.exceptions import BackupNotFound, ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from cfme.utils.wait import wait_for
+
+
+class VolumeBackupToolbar(View):
+    """The toolbar on the Volume Backup page"""
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class VolumeBackupDetailsToolbar(View):
+    """The toolbar on the Volume Backup detail page"""
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Button('Download summary in PDF format')
+
+
+class VolumeBackupDetailsEntities(View):
+    """The entities on the Volume Backup detail page"""
+    breadcrumb = BreadCrumb()
+    properties = SummaryTable('Properties')
+    relationships = SummaryTable('Relationships')
+    smart_management = SummaryTable('Smart Management')
+
+
+class VolumeBackupDetailSidebar(View):
+    """The accordion on the Volume Backup details page"""
+    @View.nested
+    class properties(Accordion):  # noqa
+        tree = ManageIQTree()
+
+    @View.nested
+    class relationships(Accordion):  # noqa
+        tree = ManageIQTree()
+
+
+class VolumeBackupView(BaseLoggedInPage):
+    """A base view for all the Volume Backup pages"""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+    flash = FlashMessages(
+        './/div[@id="flash_msg_div"]/div[@id="flash_text_div" or '
+        'contains(@class, "flash_text_div")]')
+
+    @property
+    def in_volume_backup(self):
```

Comment:
**REQUIRED:** Why is this not ```is_displayed```?

Reply:
Ahh I see why, we could always use a super in the view below...... @mfalesni what you think?

Reply:
@psav We try to avoid super to mitigate is_displayed from multiple-inheritance views which can sometimes happen and bite one's a$$ as happened quite recently, so it is better to create these `in_something` methods and use them in `is_displayed` in the child classes.

Reply:
But here should be at least is_displayed returning in_volume_backup

Reply:
@mfalesni added 'is_displayed' method returning 'in_volume_backups' but as per my understanding >  we required is_displayed for the repetitive view; this is BaseView going to inherited by other views mainly 'All'. All other main views have their owned 'is_displayed' method. It's just for checking navigation to backups.